src/routes/config.ts

```diff
@@ -0,0 +1,113 @@
+import * as _ from 'lodash';
+import { RequestHandler } from 'express';
+import { JSONSchema6, JSONSchema6Definition } from 'json-schema';
+
+import {
+	DEFAULT_SUPERVISOR_POLL_INTERVAL,
+	RESERVED_NAMES,
+	RESERVED_NAMESPACES,
+	INVALID_CHARACTER_REGEX,
+	WHITELISTED_NAMES,
+	WHITELISTED_NAMESPACES,
+	BLACKLISTED_NAMES,
+} from '../lib/env-vars';
+
+// Note that this list will be mutated by the cloud API to add
+// extra variables as for cloud-only features.
+export const SUPERVISOR_CONFIG_VAR_PROPERTIES: {
```

Comment:
I think it makes sense to move all these `const`s into `lib/env-vars` since they're really tied together.

Reply:
You're right, that makes a lot of sense, done.